server/memphis_handlers_stations.go

```diff
@@ -109,6 +109,14 @@ func validateReplicas(replicas int) error {
 	return nil
 }
 
+func validateIdempotencyWindow(retentionValue int, idempotencyWindow int) error {
+	if retentionValue > idempotencyWindow {
```

Comment:
The issue we want to check is whether the idempotency window is greater and not the opposite. So please fix your condition.
And please fix the error message to
"idempotency window cannot be greater than retention value"

Reply:
the error message is already "idempotency window cannot be greater than retention value"

Reply:
my bad I meant
"idempotency window cannot be greater than the station retention"